find_old_nonmatching.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+shopt -s globstar
+
+ls asm/non_matchings/**/*.s | sort -u -t $'\n' > old_list.txt
+grep GLOBAL_ASM -r src | cut -d '"' -f2 - | sort -u > cur_list.txt
+
+grep -F .data spec | cut -d '"' -f2 - > data_list.txt
+grep -F .rodata spec | cut -d '"' -f2 - >> data_list.txt
+grep -F .reloc spec | cut -d '"' -f2 - >> data_list.txt
+grep -F "build/asm" spec | grep -v -E 'overlays' | cut -d '"' -f2 - >> data_list.txt
+
+ls data/**/*.data.s > old_data_list.txt
+ls data/**/*.rodata.s >> old_data_list.txt
+ls data/**/*.reloc.s >> old_data_list.txt
```

Comment:
Any .s file in data/ should be removed if it's not in the spec, I don't think there is a need to only consider those.

Reply:
should be fixed now